snapm/manager/plugins/stratis.py

```
@@ -242,20 +242,35 @@ def devpath(self):
+        except DbusClientMissingPropertyError:
+            self._log_warn("Could not access filesystem.Size() property")
+            return 0
 
     @property
     def free(self):
         (pool, _) = self._get_dbus_cache()
-        size = int(pool.TotalPhysicalSize())
-        used = int(pool.TotalPhysicalUsed()[1]) if pool.TotalPhysicalUsed()[0] else 0
+        try:
+            size = int(pool.TotalPhysicalSize())
+        except DbusClientMissingPropertyError:
```

Ruff (0.15.9)

[error] Undefined name `DbusClientMissingPropertyError`

(F821)